core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -729,10 +730,43 @@ && isIntervalHourAndSecond(call)) {
       }
       writer.endList(columnListFrame);
       break;
+    case OVER:
+      unparseOver(writer, call, leftPrec, rightPrec);
+      break;
     default:
       super.unparseCall(writer, call, leftPrec, rightPrec);
     }
   }
+  private void unparseOver(SqlWriter writer, SqlCall call, final int leftPrec,
```

Comment:
please try to add a test case for the Over() clause implementation using RelNode.

Reply:
Done